gnomad_qc/v5/resources/basics.py

```diff
@@ -257,9 +257,9 @@ def get_aou_failing_genomic_metrics_samples() -> hl.Table:
 
     .. note::
 
-        Samples with low mean coverage (<30), genome coverage (<0.9), aou_hdr_coverage (<0.95),
-        or aligned_q30_bases (<0.8e11) were expected to be excluded from the AoU callset. However,
-        some such samples are still present. AoU is preparing to publish a known issue in their
+        Samples with low mean coverage (<30x), genome coverage (<90% at 20x), All of Us Hereditary Disease Risk gen
+        (AoUHDR) coverage (<95% at 20x), or aligned_q30_bases (<8e10) were expected to be excluded from the AoU
```

Comment:
we may want to confirm with their team if this is e10 or e11 given the documentation and plots don't appear to be aligned

Reply:
ah nevermind just saw your other comment